db-test/tests/db/db/query.rkt

```diff
@@ -564,7 +565,53 @@
       (with-connection c
         (check-exn exn:fail? (lambda () (query-value c "select nonsuch from notthere")))
         (check-equal? (query-value c (select-val "17"))
-                      (if (ANDFLAGS 'odbc 'issl) "17" 17))))))
+                      (if (ANDFLAGS 'odbc 'issl) "17" 17))))
+
+    ;; Added 23 May 2017.
+    ;; In certain cases malformed queries do not cause expected syntax errors to be raised
+    ;; instead they trigger a 'multiple statements given' error which is misleading.
+    ;; Fixed so correct errors are raised.
```

Comment:
I think these "Added ..." comments should not be here --- for all 3 `test-case`s

1. the tests communicate all the important info (and they're not so bad to read!)
2. it'd be better to just reference the GitHub issue https://github.com/racket/racket/issues/1702